test-infra/airflow/start_airflow_local.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python
+"""Run Airflow locally in a Docker container.
+"""
+import logging
+import os
+import subprocess
+
+if __name__ == "__main__":
+  logging.getLogger().setLevel(logging.INFO)
+  # Remove a docker container named Airflow if one exists.
+  subprocess.call(["docker", "rm", "-f", "airflow"])
```

Comment:
`check_call`?

Reply:
This is deliberate because if the container doesn't exist the command will fail but we want to keep going.